pkg/util/validation/limits.go

```diff
@@ -363,6 +365,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.Var(&l.CostAttributionLabels, costAttributionLabelsFlag, "Defines labels for cost attribution. Applies to metrics like cortex_distributor_received_attributed_samples_total. To disable, set to an empty string. For example, 'team,service' produces metrics such as cortex_distributor_received_attributed_samples_total{team='frontend', service='api'}.")
```

Comment:
How about we remove this flag? It's experimental, and we're not using it anyway.

Reply:
It is currently in-use for a handful of statically configured tenants. I think we'll need a migration period for those.